pandas/_libs/groupby_helper.pxi.in

```diff
@@ -428,14 +428,20 @@ def group_rank_{{name}}(ndarray[float64_t, ndim=2] out,
     is_datetimelike : bool
         unused in this method but provided for call compatibility with other
         Cython transformations
-    ties_method :  {'keep', 'top', 'bottom'}
-        * keep: leave NA values where they are
-        * top: smallest rank if ascending
-        * bottom: smallest rank if descending
+    ties_method : {'average', 'min', 'max', 'first', 'dense'}
+        * average: average rank of group
+        * min: lowest rank in group
+        * max: highest rank in group
+        * first: ranks assigned in order they appear in the array
+        * dense: like 'min', but rank always increases by 1 between groups
```

Comment:
This is tough to describe in one line so I'm not sure of the best way but I think it can be improved by simply changing "groups" to "values"

Reply:
https://github.com/pandas-dev/pandas/blob/5edc5c4acde1f0d05b598825ab5c6a86fe551484/pandas/core/groupby/groupby.py#L1848-L1857

Yes, I agree. It is hard to describe those methods. So I copied from there to same some time. Btw, there are some typos there too. I've raise another issue #20694